Add icon file for the Montreal AQI custom integration

normcyr opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Proposed change
<!-- 
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request.
-->
Add an icon to my newly created custom integration, which retrieves the air
quality data for the city of Montréal (Québec, Canada).

## Type of change
<!--
  What type of change does your PR introduce to the Home Assistant Brands?
  NOTE: Please, check only 1! box! 
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [x] I've added a link to my custom integration repository in the PR description
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Replace an existing icon or logo after a branding change
- [ ] Removing an icon or logo

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 
- Link to custom integration repository: https://github.com/normcyr/home-assistant-montreal-aqi

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your contribution.
-->

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

<!--
  Thank you for contributing <3
-->

frenck reviewed (changes requested):
Oh hi there @normcyr 👋 

Thanks for opening a pull request. It seem that the provided branding is different from the service provider that this integration integrates with:

![CleanShot 2025-02-09 at 23 48 23@2x](https://github.com/user-attachments/assets/31337795-53d0-47ea-9ff5-edfc2f6cf627)

Could you take a look?

../Frenck

normcyr commented:
You are right! I've added the logo of the city of Montréal to my logo. Great idea.

frenck reviewed (changes requested):
This is still a modified version of the upstream branding. We cannot accept those, as these often are violating branding guidelines. Please use the original upstream branding instead.

Thanks! 👍 

../Frenck

normcyr commented:
Alright, I replaced them with the proper branding (I hope!), and added the `dark_`options. Thank you for your patience as this is my first custom integration 🙂

frenck reviewed (approved):
Thanks, @normcyr 👍 

../Frenck
